lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb

```diff
@@ -38,6 +38,7 @@ def body_params
           # - only for HTTP POST/PATCH
           def validate_request
             return unless request.post? || request.patch?
+            return unless self.class.openapi_enabled
```

Comment:
Since this module depends on the other one, I *think* you can include it in this module so it's brought in automatically.  At present, if I merge this is will break other apps because of the dependency.